locale/en-us.yaml

```diff
@@ -1219,8 +1219,15 @@ init_description:
   other: Initialize a new project
 push_description:
   other: Push your latest changes to the platform
+push_first_new_project:
+  other: "Since this is your first time running \"push\" you'll need to create a new project on the ActiveState Platform."
```

Comment:
```suggestion
  other: "Since this is your first time running \"push\" on this project you'll need to create a new project on the ActiveState Platform."
```

Reply:
I think this is more confusing. We're talking about "This project" and then asking them to create a new one. I get what you mean, but I don't think this clarifies anything.